production/ksonnet/loki/config.libsonnet

```diff
@@ -20,7 +20,9 @@
     },
 
     ingester_client_config: {
```

Comment:
@gouthamve @Kuqd I guess this does not get applied, needs to be moved in `loki` block down below with name `ingester_client`. This does not match config schema https://github.com/grafana/loki/blob/2a8a03795adf28027bd705240dd29c9973840404/pkg/loki/loki.go#L33

Reply:
good catch yes !